src/dnf-plugins/product-id/productdb.c

```diff
@@ -55,7 +55,10 @@ void freeProductDb(ProductDb *productDb) {
 }
 
 /**
- * Read content of product db from json file into structure
+ * Read content of product db from json file into structure. The contenct of
+ * the file could look like this:
```

Comment:
minor nit pick - content is spelled incorrectly.

Reply:
Thanks. Fixed.